File: secondhandsaleadmin/src/main/java/devworks/model/base/Produktoak.java
package devworks.model.base;

public class Produktoak {
    private int id;
    private String izena;
    private String deskribapena;
    private int prezioa;
    private int idKategoria;
    private String egoera;
    private int idSaltzaile;

    public Produktoak(int id, String izena, String deskribapena, int prezioa, int idKategoria, String egoera,
            int idSaltzaile) {
        this.id = id;
        this.izena = izena;
        this.deskribapena = deskribapena;
        this.prezioa = prezioa;
        this.idKategoria = idKategoria;
        this.egoera = egoera;
        this.idSaltzaile = idSaltzaile;
    }

    public int getId() {
        return id;
    }

    public String getIzena() {
        return izena;
    }

    public String getDeskribapena() {
        return deskribapena;
    }

    public int getPrezioa() {
        return prezioa;
    }

    public int getIdKategoria() {
        return idKategoria;
    }

    public String getEgoera() {
        return egoera;
    }

    public int getIdSaltzaile() {
        return idSaltzaile;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setIzena(String izena) {
        this.izena = izena;
    }

    public void setDeskribapena(String deskribapena) {
        this.deskribapena = deskribapena;
    }

    public void setPrezioa(int prezioa) {
        this.prezioa = prezioa;
    }

    public void setIdKategoria(int idKategoria) {
        this.idKategoria = idKategoria;
    }

    public void setEgoera(String egoera) {
        this.egoera = egoera;
    }

    public void setIdSaltzaile(int idSaltzaile) {
        this.idSaltzaile = idSaltzaile;
    }

    @Override
    public String toString() {
        return "Produktuak{" +
                "id=" + id +
                ", izena='" + izena + '\'' +
                ", deskribapena='" + deskribapena + '\'' +
                ", prezioa=" + prezioa +
                ", idKategoria=" + idKategoria +
                ", egoera='" + egoera + '\'' +
                ", idSaltzaile=" + idSaltzaile +
                '}';
    }
}

File: phython/Produktoak.py
class Produktoak():
    def __init__(self, ida, izena, deskribapena, prezioa, idkategoria, egoera, idbezeroa, salduta):
        self.IDAK = ida
        self.IZENAK = izena
        self.DESKRIBAPENA = deskribapena
        self.PREZIOA = prezioa
        self.IDKATEGORA = idkategoria
        self.EGOERA = egoera
        self.IDBEZEROA = idbezeroa
        self.SALDUTA = salduta

    def getIda(self):
        return self.IDAK

    def setAtributte(self):
        self.IDAK = input("Id sartu: ")
        self.IZENAK = input("Izena sartu: ")
        self.DESKRIBAPENA = input("deskribapena sartu: ")
        self.PREZIOA = input("prezioa sartu: ")
        self.IDKATEGORA = input("idkategoria sartu: ")
        self.EGOERA = input("egoera sartu: ")
        self.IDBEZEROA = input("idbezeroa sartu: ")
        self.SALDUTA = input("salduta dago?: ")

    def __setid(self):
        self.IDAK = input("ida definitu: ")

    def __setizena(self):
        self.IZENAK = input("izena definitu: ")

    def __setdeskribapena(self):
        self.DESKRIBAPENA = input("deskribapena definitu: ")

    def __setprezioa(self):
        self.PREZIOA = input("prezioa definitu: ")

    def __setidkategoria(self):
        self.IDKATEGORA = input("idkategoria definitu: ")

    def __setegoera(self):
        self.EGOERA = input("egoera kodea definitu: ")

    def __setidbezeroa(self):
        self.IDBEZEROA = input("idbezeroa definitu: ")

    def __setsalduta(self):
        self.SALDUTA = input("salduta definitu: ")



    def __getid(self):
        return self.IDAK

    def __getizena(self):
        return self.IZENAK

    def __getdeskribapena(self):
        return self.DESKRIBAPENA

    def __getprezioa(self):
        return self.PREZIOA

    def __getidkategoria(self):
        return self.IDKATEGORA

    def __getegoera(self):
        return self.EGOERA

    def __getidbezeroa(self):
        return self.IDBEZEROA

    def __getsalduta(self):
        return self.SALDUTA


    def print(self):
        print(self.IDAK, self.IZENAK, self.DESKRIBAPENA, self.PREZIOA, self.IDKATEGORA, self.EGOERA, self.IDBEZEROA, self.SALDUTA, end=' ')